cloudify_marketplace/provision/bootstrap_simple

```diff
@@ -27,11 +29,11 @@ function create_config
   echo "manager:
   private_ip: 127.0.0.1
   public_ip: 127.0.0.1
-  admin_username: ${CLOUDIFY_MANAGER_USERNAME}
-  admin_password: ${CLOUDIFY_MANAGER_PASSWORD}
+  admin_username: ${CLOUDIFY_MANAGER_USERNAME:-admin}
```

Comment:
i'm not sure how that works, won't the env var be set to admin anyway? based on cloudify_marketplace/cloudify.json?

Reply:
I just added this here so that it would be very simple to just take this whole script it and run it in order to locally install a manager (with minimal adjustments; e.g. passing the install RPM link)